scripts/submit_env.sh

```diff
@@ -0,0 +1,138 @@
+#!/bin/bash
+
+save_env() {
+    # save the current environment to the file startup_environment.sh 
+    # this is intended to be the first function run by 
+    # - gWMS-CMSRunAnalysis.sh: when running a job on the global pool
+    # - crab preparelocal, crab submit --dryrun: when running a job locally
+
+    # On some sites we know there was some problems with environment cleaning
+    # with using 'env -i'. To overcome this issue, whenever we start a job, we have
+    # to save full current environment into file, and whenever it is needed we can load
+    # it. Be aware, that there are some read-only variables, like: BASHOPTS, BASH_VERSINFO,
+    # EUID, PPID, SHELLOPTS, UID, etc.
+
+    # Moreover, src/python/WMCore/Storage/Backends/GFAL2Impl.py 
+    # makes use of startup_environment.sh
+
+    export DMDEBUGVAR=dmdebugvalue-env-cmsrunanalysis
+    export JOBSTARTDIR=$PWD
+    export HOME=${HOME:-$PWD}
+
+    declare -p > startup_environment.sh
+
+    # these lines are for debugging purposes only: start 
+    echo "DM DEBUG: cat startup_environment.sh"
+    # basename -- "$0"
+    # dirname -- "$0"
+    # echo $PWD
+    # ls -lrth
+    cat startup_environment.sh
```

Comment:
my question is: do we want to keep this `cat`?

Reply:
no.

Reply:
only you can judge if it is worth to have it in production for a week and then remove.

Reply:
I will remove it, then. 

In case of problems, we can always use a scriptExe that dumps that file. Or we can think about copying it back to the schedd

Reply:
I removed the `cat` and some other debugging prints. I added to this comment https://github.com/dmwm/CRABServer/issues/7084#issuecomment-1192371547 a note about copying the file `start_environment.sh` back to schedd so that i dont forget about this idea.

I think this PR is now ready.